app/services/proofing/lexis_nexis/ddp/response_redacter.rb

```diff
@@ -197,12 +197,7 @@ class ResponseRedacter
         def self.redact(hash)
           return { error: 'TMx response body was empty' } if hash.nil?
           return { error: 'TMx response body was malformed' } unless hash.is_a? Hash
-          filtered_response_h = hash.slice(*ALLOWED_RESPONSE_FIELDS)
-          unfiltered_keys = hash.keys - filtered_response_h.keys
```

Comment:
I briefly had the thought that we could keep this, and then do something like:

```ruby
filtered_response_h['redacted_keys'] = unfiltered_keys.join(", ")
```

I don't see a need for this, though.